### ISSUE TITLE
no device recognized

### ISSUE THREAD

**Issue by NEWxGAME** [DarthAffe/RGBSyncPlus#9]
At first i take RGBSync+, i unlock before extracting, and the program close directly. After i compile myself and now no deviced recognized, do you have any solution ?

PS: sorry for my english

**Comment by NEWxGAME** [comment]
thanks for your reply.
i Don't have a file error.log but when a remove the auraSDK RGBsyncplus work and i have device but not the asus devices, i think your right is the auraSDK crash the program. And i have the same problem now when i build myself. Do you have any solution for fix the problem?

**Comment by DarthAffe** [comment]
no, i've created a forum-post about this quite some time ago but until now there wasn't anything new regarding that problem (https://rog.asus.com/forum/showthread.php?97754-Access-Violation-Wrong-EnumerateMB-Result)

**Comment by NEWxGAME** [comment]
Perhaps this help you:
I go to window event error and i have 2 error for the crash of the program:

first error:

Application : RGBSync+.exe
Version du Framework : v4.0.30319
Description : le processus a été arrêté en raison d'une exception non gérée.
Informations sur l'exception : System.AccessViolationException
   à RGB.NET.Devices.Asus.Native._AsusSDK.SetClaymoreKeyboardMode(IntPtr, Int32)
   à RGB.NET.Devices.Asus.AsusDeviceProvider.Initialize(RGB.NET.Core.RGBDeviceType, Boolean, Boolean)
   à RGB.NET.Core.RGBSurface.LoadDevices(RGB.NET.Core.IRGBDeviceProvider, RGB.NET.Core.RGBDeviceType, Boolean, Boolean)
   à RGB.NET.Core.RGBSurface.LoadDevices(RGB.NET.Core.IRGBDeviceProviderLoader, RGB.NET.Core.RGBDeviceType, Boolean, Boolean)
   à RGBSyncPlus.ApplicationManager.LoadDeviceProviders()
   à RGBSyncPlus.ApplicationManager.Initialize()
   à RGBSyncPlus.App.OnStartup(System.Windows.StartupEventArgs)
   à System.Windows.Application.<.ctor>b__1_0(System.Object)
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   à System.Windows.Threading.DispatcherOperation.InvokeImpl()
   à System.Windows.Threading.DispatcherOperation.InvokeInSecurityContext(System.Object)
   à System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   à System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   à System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   à MS.Internal.CulturePreservingExecutionContext.Run(MS.Internal.CulturePreservingExecutionContext, System.Threading.ContextCallback, System.Object)
   à System.Windows.Threading.DispatcherOperation.Invoke()
   à System.Windows.Threading.Dispatcher.ProcessQueue()
   à System.Windows.Threading.Dispatcher.WndProcHook(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   à MS.Win32.HwndWrapper.WndProc(IntPtr, Int32, IntPtr, IntPtr, Boolean ByRef)
   à MS.Win32.HwndSubclass.DispatcherCallbackOperation(System.Object)
   à System.Windows.Threading.ExceptionWrapper.InternalRealCall(System.Delegate, System.Object, Int32)
   à System.Windows.Threading.ExceptionWrapper.TryCatchWhen(System.Object, System.Delegate, System.Object, Int32, System.Delegate)
   à System.Windows.Threading.Dispatcher.LegacyInvokeImpl(System.Windows.Threading.DispatcherPriority, System.TimeSpan, System.Delegate, System.Object, Int32)
   à MS.Win32.HwndSubclass.SubclassWndProc(IntPtr, Int32, IntPtr, IntPtr)
   à MS.Win32.UnsafeNativeMethods.DispatchMessage(System.Windows.Interop.MSG ByRef)
   à System.Windows.Threading.Dispatcher.PushFrameImpl(System.Windows.Threading.DispatcherFrame)
   à System.Windows.Threading.Dispatcher.PushFrame(System.Windows.Threading.DispatcherFrame)
   à System.Windows.Application.RunDispatcher(System.Object)
   à System.Windows.Application.RunInternal(System.Windows.Window)
   à System.Windows.Application.Run(System.Windows.Window)
   à System.Windows.Application.Run()
   à RGBSyncPlus.App.Main()

and second error:
Nom de l’application défaillante RGBSync+.exe, version : 1.0.0.0, horodatage : 0x5b952db2
Nom du module défaillant : unknown, version : 0.0.0.0, horodatage : 0x00000000
Code d’exception : 0xc0000005
Décalage d’erreur : 0x23c1ca23
ID du processus défaillant : 0xb30
Heure de début de l’application défaillante : 0x01d47918408d9fe0
ID de rapport : fa0c578c-1ba3-4c7a-abbb-728159c826f8

**Comment by DarthAffe** [comment]
This is exactly this problem. Right now I don't know of any way to get around this.

**Comment by NEWxGAME** [comment]
I find the materiel to create this problem, the problem is the keyboard rog claymor core.

**Comment by DarthAffe** [comment]
Oh ok, that's interesting. If the problem doesn't persist if you unplug the keyboard it might be a bug related to the keyboard-implementation. I'll take a look at it.

**Comment by NEWxGAME** [comment]
Hello,

Do you find the solution for the problem with the keyboard?

**Comment by DarthAffe** [comment]
No sorry, I've no idea what could cause that issue (aside from the known issue with the sdk). Maybe if you (or someone else) is able to verify if the keyboard SDK is working with the official examples we would at least know that for sure.

**Comment by D0N2k** [comment]
What can I do to help?

**Comment by NEWxGAME** [comment]
I have a problem with the keyboard rog claymore and when i unplug this keyboard the application function, if i have the Time i test with the official exemple

**Comment by D0N2k** [comment]
I tried the AURA_SDK_Test.exe thats contained within the AuraSDK from the asus site and the colorchange with the Claymore Core works.
